api/src/main/java/io/opencensus/metrics/export/Distribution.java

```diff
@@ -163,6 +130,99 @@ public static Distribution create(
    */
   public abstract List<Bucket> getBuckets();
 
+  /**
+   * The bucket boundaries used to create a histogram for the distribution.
+   *
+   * @since 0.17
+   */
+  @AutoValue
+  @Immutable
+  public abstract static class BucketOptions {
+
+    BucketOptions() {}
+
+    /**
+     * Creates a {@link BucketOptions}.
+     *
+     * @param explicitBuckets the explicit buckets with arbitrary widths.
+     * @return a {@code BucketOptions}.
+     * @since 0.17
+     */
+    public static BucketOptions create(Explicit explicitBuckets) {
+      Utils.checkNotNull(explicitBuckets, "explicitBuckets");
+      return new AutoValue_Distribution_BucketOptions(explicitBuckets);
+    }
+
+    /**
+     * Returns the {@link Explicit} associated with the {@link BucketOptions}.
+     *
+     * @return the {@code Explicit} associated with the {@code BucketOptions}.
+     * @since 0.17
+     */
+    public abstract Explicit getExplicitBuckets();
+  }
+
+  /**
+   * A Bucket with explicit bounds.
```

Comment:
`Explicit` is a list of bucket bounds instead of one bucket. Maybe change to "A list of buckets with arbitrary widths."?